examples/LottieExtension.cpp

```diff
@@ -24,71 +24,105 @@
 /* Drawing Commands                                                     */
 /************************************************************************/
 
-static unique_ptr<tvg::LottieAnimation> animation;
-static Elm_Transit *transit;
-static bool updated = false;
+static unique_ptr<tvg::LottieAnimation> slotAnimation;
+static unique_ptr<tvg::LottieAnimation> markerAnimation;
+static Elm_Transit *slotTransit;
+static Elm_Transit *markerTransit;
+static int updated = 0;
 
 void tvgUpdateCmds(tvg::Canvas* canvas, tvg::LottieAnimation* animation, float progress)
 {
-    if (updated || !canvas) return;
+    if (updated >= 2 || !canvas) return;
```

Comment:
We may wanna avoid hard coded condition.
You can define constant value in top of file and reuse them.
```cpp
#define NUM_TOTAL_UPDATE 2
if (updated >= NUM_TOTAL_UPDATE || !canvas) return;
```

just FYI. we don't need to use `updated` condition, please find out `examples/Lottie.cpp`